pytorch/training/docker/1.9/py3/example/Dockerfile.gpu

```diff
@@ -0,0 +1,12 @@
+
+# Use the Deep Learning Container as a base Image
+ARG BASE_IMAGE=""
+
+FROM $BASE_IMAGE
+
+RUN pip install tensorboardX==1.6.0
```

Comment:
Is this still 1.6 as per PT requirements?

Reply:
PyTorch does not list tensorboardX compatibility requirement. There is some depedency of on tensoboard requirement [here](https://github.com/pytorch/pytorch/blob/4b96fc060b0cb810965b5c8c08bc862a69965667/torch/utils/tensorboard/__init__.py#L6), but checks for minimum version. We can upgrade tensorboarX package to 2.2 .